src/Modals.tsx

```diff
@@ -96,7 +97,7 @@ export const Modals = () => {
 
       {shouldShowPWAModal && <PwaModal />}
 
-      {isNotificationPwaModalOpen && <NotificationPwaModal />}
+      {!isCI && isNotificationPwaModalOpen && <NotificationPwaModal />}
```

Comment:
I think we can avoid this flag by [granting the `notification` permission](https://playwright.dev/docs/api/class-browsercontext#browser-context-grant-permissions). Also we'll need to test these states anyway so this will have to change.